code/modules/mob/living/simple_animal/hostile/mining_mobs.dm

```diff
@@ -833,8 +833,8 @@
 	desc = "[src] has become inert, it crackles no more and is useless for \
 		healing injuries."
 
-/obj/item/organ/hivelord_core/legion/preserved()
-	..()
+/obj/item/organ/hivelord_core/legion/preserved(implanted = 0)
+	..(implanted)
```

Comment:
do I actually need to pass this through? or is this call special in that it automatically picks up the passed in vars?